### ISSUE TITLE
Transcript - Content issues - Jan 18

### ISSUE THREAD

**Issue by shosein** [EduShareOntario/studentTranscriptOCASIntegration#27]
[Content - Transcript Review - updated Jan 18-2016.xlsx](https://github.com/EduShareOntario/studentTranscriptOCASIntegration/files/94440/Content.-.Transcript.Review.-.updated.Jan.18-2016.xlsx)

Transcript review - content issues outstanding - please see all worksheets attached.

**Comment by shosein** [comment]
1. [New content.pdf](https://github.com/EduShareOntario/studentTranscriptOCASIntegration/files/94657/New.content.pdf)
These samples match the students in the excel spreadsheet.  It provides guidance on where to add the new content.

2. In the excel spreadsheet, we had listed for 4 low issues.  we have decided we do not need to worry about these. 
![disregard these issues](https://cloud.githubusercontent.com/assets/15947632/12396550/060ce05a-bdd6-11e5-8687-5123d24f8932.JPG)
